src/lib/data/schema.ts

```diff
@@ -447,8 +480,19 @@ export const usersRelations = relations(users, ({ many }) => ({
   tagScores: many(userTagScores),
   dailySummaries: many(userSummaries),
   dailyScores: many(userDailyScores),
+  walletAddresses: many(walletAddresses),
 }));
 
+export const walletAddressesRelations = relations(
+  walletAddresses,
+  ({ one }) => ({
+    user: one(users, {
+      fields: [walletAddresses.userId],
+      references: [users.username],
+    }),
+  }),
+);
+
```

Comment:
Would probably be more useful to have relation from user to many walletAddresses

Reply:
Ah my bad I missed that, all good